Improve substring function entry (#4527)
mendix/docs: content/en/docs/refguide/modeling/application-logic/expressions/string-function-calls.md

| ------------------------------------------------------------ | ------ |
| A part of the original string, starting at the start position with a length equal to the desired length. If no desired length is specified, will return a substring starting at the start position and ending at the end of the string. | String |

{{% alert color="warning" %}}
The function will output an error for the following:

> * When the start position of the substring is larger than the position of the last character

Reviewer: ?
```suggestion
* When the start position of the substring is after the last character in the string
```

Author: Thank you for making it simpler @MarkvanMents

> * When the desired length of the result is larger than the length of the substring

Reviewer: ```suggestion
* When the desired length of the result is longer than the substring
```
